src/main/java/frc/robot/RobotMap.java

```diff
@@ -14,9 +14,9 @@
     public static final int[] RIGHT_ENCODER_PORTS = {2, 3}; 
 
     // PID
-    public static final double Kp = 0.0001;
-    public static final double Ki = 0.0001;
-    public static final double Kd = 0.0001;
+    public static final double DRIVETRAIN_KP = 0.0001;
+    public static final double DRIVETRAIN_KI = 0.0001;
+    public static final double DRIVETRAIN_KD = 0.0001;
```

Comment:
KD is probably going to be negative, but double check the PIDController docs that they aren't already assuming that and multiplying whatever you give them by -1

Reply:
Said nothing in the docs, and looking at the source code looks like they don't multiply by -1